injector_test.go

```diff
@@ -408,3 +427,20 @@ func BenchmarkProcess(b *testing.B) {
 		}
 	})
 }
+
+func assertJSONEqual(t *testing.T, expected, actual string) {
+	var expectedJSON, actualJSON interface{}
+
+	if err := json.Unmarshal([]byte(expected), &expectedJSON); err != nil {
+		t.Errorf("Expected ('%s') is not a valid json: '%s'", expected, err.Error())
+	}
+
+	if err := json.Unmarshal([]byte(actual), &actualJSON); err != nil {
+		t.Errorf("Actual ('%s') is not a valid json: '%s'", actual, err.Error())
```

Comment:
- `Fatalf`
- `err` в аргументах не нуждается в вызове `Error()`
- ```suggestion
		t.Fatalf("Actual is not a valid json: '%s'", err)
```
и ниже лишняя строка

Reply:
Готово